vignettes/tcplfit2-vignette.Rmd

```diff
@@ -161,3 +161,147 @@ The input data for this example contains 6 signatures for one chemical in a tran
   }
 ```
 
+## Example 4: Running tcplfit2 on standalone tcpl-like data from mc0.
+
+The input for this example is from ACEA data that has been released in past versions of the ToxCast database, invitrodb. Those familiar with the data format in the ToxCast Pipeline (tcpl) will recognize the data in the format of multi-concentration level 0 data otherwise known as mc0.  Data from the assay component ACEA_AR_agonist_80hr was analyzed in the positive analysis fitting direction relative to DMSO as the neutral control and baseline of activity. Using a electrical impedance as a cell growth reporter, increased activity can be used to infer increased signaling at the pathway-level for the androgen receptor (as encoded by the AR gene).
+
+```{r example4_init, fig.height = 6, fig.width = 7}
+
+load(file = "../data/mc0.rda")
+library(tcpl)
+library(data.table)
+dat <- mc0
+DT::datatable(head(dat),options = list(scrollX = T))
+```
+
+The above table is identical to the mc0 table that would be seen in invitrodb and recognized by tcpl.  However, some users may want to run standalone tcplfit2 fitting without the need for a MySQL database connection to invitrodb.  The first steps (levels 1 through 3 in tcpl) are used to adjust the data and determine the baseline sampling variability, or noise, that will later be used for cutoff calculation. For further information on normalization and determination of a cutoff, see vignettes for library(tcpl) at https://CRAN.R-project.org/package=tcpl .
+```{r example4_mc2, fig.height = 6, fig.width = 7}
+#In this case there are no adjustments required for the data so we set cval to the original rval
+dat[,cval := rval]
+## Fitting generally cannot occur if response values are NA or null so we remove them
+dat <- dat[!is.na(cval),]
+## Add column for log10 concentration as some of the mc3 methods require logc
+dat[ , logc := log10(conc)]
+
+## initialize dummy aeid required if using mc3_mthds from tcpl
+dummy_aeid <- 99999
+dat[,aeid := dummy_aeid]
+
+## Set aeid as a key
+setkey(dat,aeid)
+```
+
+Once the data is initialized to a point where the required fields are available we can use the methods included in the tcpl package without the need for a database connection.  You can see the list of available methods for mc3 in the table below.
```

Comment:
The section above (Lines 64 - 194) could benefit from the use of backticks and other formatting to highlight key terms like invitrodb, tcplfit2, etc. It appears that this style was applied to Examples 1 - 3 and Line 200 onward, but not the first half of example 4.

Reply:
I tried to add some highlights let me know and/or add them in areas if you think i missed any

Reply:
Great!  on review, I noticed that Line 64: invitrodb is written as `invitrodb`, but lines 166, lines 177 (listed twice) are not.  I don't have a style preference, but making those occurrence match might be nice.